docs/minimal/transformations.md

```diff
@@ -0,0 +1,53 @@
+|-------------------------|-------------------------------------|-----------------------|----------------------------------------------------------------------------------------|
+| TINYINT                 | Long                              | INT64     | Long                                                                                 |
+| INT                     | Long                              | INT64     | Long                                                                                 |
+| BIGINT                  | Long                              | INT64    | Long                                                                                 |
+| TIME                    | Long ([time-micros](https://avro.apache.org/docs/current/specification/_print/#time-microsecond-precision) ex: 45296000000 for 12:34:56) | STRING      | Long|
+| YEAR                    | Long                              | STRING     | Long                                                                                 |
+| FLOAT                   | Double                          | FLOAT32     | Double                                                                                 |
+| DOUBLE                  | Double                          | FLOAT64     | Double                                                                                 |
+| DECIMAL                 | String                              | NUMERIC      | String                                                                                 |
+| BOOLEAN                 | Long                | BOOLEAN     | Long                                                                                 |
+| TEXT                    | String                              | STRING     | String                         |
+| ENUM                    | String                              | STRING     | String                               |
+| BLOB                    | String (hex encoded)             | BYTES     | Binary String                                                                          |
+| BINARY                  | String (hex encoded)             | BYTES     | Binary String                                                                          |
+| BIT                     | Long             | BYTES     | Long                                                                          |
+| DATE                    | String (Format: yyyy-MM-dd))        | DATE     | String( Format: yyyy-MM-dd)            |
+| DATETIME                | String (ex: 2024-01-01T12:34:56Z)   | TIMESTAMP     | String                                                                                 |
+| TIMESTAMP               | String (ex: 2024-01-01T12:34:56Z)   | TIMESTAMP     | String                                                                                 |
+
```

Comment:
It seems like we are missing certain types for mysql:

This is a fuller list here:
https://cloud.google.com/datastream/docs/unified-types#map-mysql

This doc still misses unsupported spatial types:
"spatial_linestring",
            "spatial_multilinestring",
            "spatial_multipoint",
            "spatial_multipolygon",
            "spatial_point",
            "spatial_polygon"

For these we can just say expect null since not supported.

Reply:
Added